src/dvui.zig

```diff
@@ -2506,7 +2506,7 @@ pub const Window = struct {
 
         pub fn free(self: *const SavedData, allocator: std.mem.Allocator) void {
             if (self.data.len != 0) {
-                allocator.rawFree(self.data, @ctz(self.alignment), @returnAddress());
+                allocator.rawFree(self.data, @enumFromInt(@ctz(self.alignment)), @returnAddress());
```

Comment:
Tbh memory alignment is way over my head, so please correct me if this "fix" is wrong...